config/pimcore/execution_engine.yaml

```diff
@@ -0,0 +1,12 @@
+pimcore_generic_execution_engine:
+  error_handling: 'continue_on_error'
```

Comment:
I would say we should change this to `stop_on_first_error` :)

Reply:
Done :)